app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,5 @@
+class BlogsController < ApplicationController
+  def index
```

Comment:
Please add `users_controller` and `posts_controller` with #index and #show actions. Moreover, please add corresponding view files.

Reply:
user_controller, blog_controller added with index and show methods respectively